app/models/event.py

```diff
@@ -447,6 +447,9 @@ def has_sessions(self):
     def has_speakers(self):
         return Speaker.query.filter_by(event_id=self.id).count() > 0
 
+    @property
+    def order_statistics(self):
+        return dict
```

Comment:
Wow! What use is this for? You have to return the actual order statistics from here!

Reply:
You mean to return `dict['order_statistics']`